cookbook/form/data_transformers.rst

```diff
@@ -106,10 +97,10 @@ in your code.
 You can also add the transformer, right when adding the field by changing the format
 slightly::
 
-    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
 
     $builder->add(
-        $builder->create('description', TextareaType::class)
+        $builder->create('description', TextType::class)
```

Comment:
removing the `TextType`means i need to remove the `data_class`in the example too because else it would not be clear anymore which Type it is.